### DISCUSSION TITLE
0.9.20 build fail

### DISCUSSION THREAD

**Discussion by DreamMaoMao** [Alexays/Waybar#2327] [General]
![image](https://github.com/Alexays/Waybar/assets/30348075/cc2bccd3-d74b-43f4-b4ca-1adbbbf68677)
![image](https://github.com/Alexays/Waybar/assets/30348075/898d8438-fa15-4633-adc0-bde4d1b4d0a2)

Can anyone tell me how to solve this problem please?  Is the version of the compiler required?

**Comment by LukashonakV** [comment]
Fixed at #2328. Found at #2281

**Reply by DreamMaoMao** [reply]
it work to me  thanks you!
